app/common/validation/PropertyValidation.scala

```diff
@@ -48,4 +48,20 @@ object PropertyValidation {
         broughtForwardLosses, annualExemptAmount, disposalDate)))
     }
   }
+
+  def validatePropertyTaxOwed(propertyCalculateTaxOwedModel: PropertyCalculateTaxOwedModel): Either[String, PropertyCalculateTaxOwedModel] = {
+    val propertyChargeableGainModel = validatePropertyChargeableGain(propertyCalculateTaxOwedModel.propertyChargeableGainModel)
+    val previousTaxableGain = validateOptionDouble(propertyCalculateTaxOwedModel.previousTaxableGain, residentPropertyKeys.previousTaxableGain)
+    val previousIncome = validateDouble(propertyCalculateTaxOwedModel.previousIncome, residentPropertyKeys.previousIncome)
+    val disposalDate = validateDisposalDate(propertyCalculateTaxOwedModel.propertyChargeableGainModel.disposalDate)
```

Comment:
This validation is already done in the validation for the Chargeable Gain so I don't think you need it here at all.